pgrx-examples/bad_ideas/src/lib.rs

```diff
@@ -113,7 +113,7 @@ fn random_abort() {
 pub unsafe extern "C" fn _PG_init() {
     #[pg_guard]
     extern "C" fn random_abort_callback(event: pg_sys::XactEvent, _arg: *mut std::os::raw::c_void) {
-        // info!("in global xact callback: event={}", event);
+        // info!("in global xact callback: event={event}");
```

Comment:
??? why is this line even here?

Reply:
someone was debugging something?  i can delete it if you want